certbot/compat/os.py

```diff
@@ -29,3 +29,15 @@
 
 # Clean all remaining importables that are not from the core os module.
 del ourselves, std_os, std_sys
+
+
+# The os.open function on Windows will have the same effect than a bare os.chown towards the given
+# mode, and will create a file with the same flaws that what have been described for os.chown.
+# So upon file creation, security.take_ownership will be called to ensure current user is the owner
+# of the file, and security.chmod will do the same thing than for the modified os.chown.
+# Internally, take_ownership will update the existing metdata of the file, to set the current
+# username (resolved thanks to win32api module) as the owner of the file.
```

Comment:
```suggestion
# username (determined by the win32api module) as the owner of the file.
```

Reply:
Also presumably all the mentions of `security` in this explanation should be `filesystem`.